gui/storage/models.py

```diff
@@ -1111,14 +1111,26 @@ def get_password(self):
         return notifier().pwenc_decrypt(self.password)
 
 
-class QuotaExcess(Model):
+class RootDataset(Model):
+    quota_warning = models.FloatField(verbose_name=_('Quota warning at, %'), help_text=_('0=Disabled'))
+    quota_critical = models.FloatField(verbose_name=_('Quota critical alert at, %'), help_text=_('0=Disabled'))
+    refquota_warning = models.FloatField(verbose_name=_('Refquota warning at, %'), help_text=_('0=Disabled'))
+    refquota_critical = models.FloatField(verbose_name=_('Refquota critical alert at, %'), help_text=_('0=Disabled'))
```

Comment:
Why float? Do we really need that kind of granularity? 77.52%?

Reply:
@william-gr Because I wanted value from 0 to 1.